### PR TITLE
View/player interface enhancements

### PR DESCRIPTION
**Pull request by MelissaRoman06**
Player interface enhancements. New icons and visually better.

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 1]
```diff
@@ -57,19 +56,19 @@
      */
     public PlayerJFrame(String path) {
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: Delete this empty line

Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: Remove this space.

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 2]
```diff
@@ -57,19 +56,19 @@
...
+        pauseButton = new JButton();
         playingPanel = new JPanel();
         bottomPanel = new JPanel();
         progressBar = new JSlider(0, 100, 0);
-        volumen = new JSlider(0, 100, vol);
+        volumeLevel = 50;
+        volumeSlider = new JSlider(JSlider.VERTICAL,0, 100, volumeLevel);
+        //JSlider slider = new JSlider(JSlider.VERTICAL, 0, 20, 4);
         player = new EmbeddedMediaPlayerComponent();
         fileToBePlayed = new File(path);
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java:68]: remove this line.

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 3]
```diff
@@ -118,7 +100,7 @@ public PlayerJFrame(String path) {
         setVisible(true);
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java:98]: remove this line.

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 4]
```diff
@@ -57,19 +56,19 @@
...
-        volumen = new JSlider(0, 100, vol);
+        volumeLevel = 50;
+        volumeSlider = new JSlider(JSlider.VERTICAL,0, 100, volumeLevel);
+        //JSlider slider = new JSlider(JSlider.VERTICAL, 0, 20, 4);
         player = new EmbeddedMediaPlayerComponent();
         fileToBePlayed = new File(path);
 
-        setBounds(new Rectangle(80, 100, 800, 600));
+        setBounds(new Rectangle(80, 100, 800, 540));
         setTitle("LogicalPlayer");
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: remove this line.

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 5]
```diff
@@ -80,35 +79,18 @@ public PlayerJFrame(String path) {
             System.out.println("Icons cannot be found..");
         }
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java:78]: Remove this line.

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 6]
```diff
@@ -33,15 +33,14 @@
  */
 public class PlayerJFrame extends JFrame {
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: delete this empty line

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 7]
```diff
@@ -57,19 +56,19 @@
...
-        headerPanel = new JPanel();
+        playButton = new JButton();
+        stopButton = new JButton();
+        pauseButton = new JButton();
         playingPanel = new JPanel();
         bottomPanel = new JPanel();
         progressBar = new JSlider(0, 100, 0);
-        volumen = new JSlider(0, 100, vol);
+        volumeLevel = 50;
+        volumeSlider = new JSlider(JSlider.VERTICAL,0, 100, volumeLevel);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: put space after of  "," on 0

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 6 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 7 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java:68]: The line is used to separate the declaration part from the rest of the code

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java:98]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 5 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java:78]: the line is to separate the code

**Pull request merged**